Source/VMware.vSphereDSC/VMware.vSphereDSC.build.ps1

```diff
@@ -29,6 +29,40 @@ $script:LicenseFileContent = Get-Content -Path $script:LicensePath | Select-Obje
 $script:ImportFolders = @('Enums', 'Classes', 'DSCResources')
 $script:DSCResourcesFolder = Join-Path -Path $script:ModuleRoot -ChildPath "DSCResources"
 
+function Get-LinesRange {
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param(
+        [System.Object[]] $FileContent,
+        [string] $StartLinePattern,
+        [string] $EndLinePattern
+    )
+
+    $range = @{}
+
+    $index = 1
+    $startLine = 0
+    $endLine = 0
+
+    foreach ($line in $FileContent) {
+        if ($line -Like $StartLinePattern) {
+            $startLine = $index
+        }
+
+        if ($line.Trim().EndsWith($EndLinePattern) -and $startLine -ne 0) {
+            $endLine = $index
+            break
+        }
+
+        $index++
+    }
+
+    $range.Set_Item("startLine", $startLine)
```

Comment:
Could be replaced with

$range.StartLine = $startLine
$range.EndLine = $endLine